src/components/Navbar/Navbar.jsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
```

Comment:
since we're not going to be using a lot of icons, I recommend an implementation without additional libraries, but adding a few SVGs to our assets directory